src/main/antlr/LibSLLexer.g4

```diff
@@ -272,19 +309,43 @@ CHARACTER
    |   '\'' EscapeSequence '\''
    ;
 
-fragment
-SingleCharacter
+fragment SingleCharacter
     :   ~['\\\r\n]
     ;
 
-fragment
-EscapeSequence
-    :   '\\u' Hex Hex Hex Hex Hex Hex Hex Hex
+fragment EscapeSequence
+    :   '\\' 'u005c'? [btnfr"'\\]
```

Comment:
What is the purpose of this expression?
What is `\u005cb` ? Why do we need to explicitly support it?

Reply:
Fixed;